pymongo/client_session.py

```diff
@@ -1000,6 +1009,24 @@ def __copy__(self):
         raise TypeError("A ClientSession cannot be copied, create a new session instead")
 
 
+class _EmptyServerSession(object):
+    def __init__(self, generation):
+        self.generation = generation
+        self.last_use = time.monotonic()
+        self.lsid = None
+        self.dirty = False
+        self.started = 0
+
+    def timed_out(self, session_timeout_minutes):
+        return True
+
+    def mark_dirty(self):
+        self.dirty = True
+
+    def inc_transaction_id(self):
+        self.started += 1
```

Comment:
`old.started` should just be a boolean since inc_transaction_id() will only be called once. All that matters is wether we should forward the inc_transaction_id() call once when the session is materialized.

Reply:
Done.